libraries/Microsoft.Bot.Builder.Azure/Microsoft.Bot.Builder.Azure.csproj

```diff
@@ -10,6 +10,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Microsoft.Bot.Builder.Azure</PackageId>
```

Comment:
These changes should be part of the PR's description

Reply:
Added to the description 👍🏻